test/robot/replay/replay.proto

```diff
@@ -44,8 +44,8 @@ message Input {
   string virtual_swap_chain_json = 8;
   // Package is the stash id of the package used to generate the report.
   string package = 9;
-  // API is the API of the commands to replay in the trace.
-  string API = 10;
+  // Api is the API of the commands to replay in the trace.
+  string Api = 10;
```

Comment:
Again, why is `Api` titled when all other fields are all lowercase?

Reply:
Fixed in 1613802